config.json.example

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
How about, instead of adding an example file to the project root directory, we maintain the template (perhaps as a Python object of type `List[Dict]`) within `gentest.py` (for now) that we dump to `~/eest/gentest.json` upon request? This could be achieved via `gentest --init-config`? This seems even lower overhead for new users to get started?

Reply:
We can leverage Jijna for templating config too. I can either do it in this PR or as part of the #863.